src/classes/database.ts

```diff
@@ -78,141 +101,126 @@ export class Database {
-          "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Anomaly.points": true,
-          "data.DDT_AllLoadoutsBit.teenAffinities": true,
-          "data.DDT_AllLoadoutsBit.charXpLevelCosts": true,
-          "data.DDT_AllWeaponsBit.pointsByWeaponType": true,
-          "data.DDT_AllWeaponsBit.teenWeaponUnlockLevels": true,
-          "data.DDT_AllWeaponsBit.weaponXpToNextLevel": true,
-          "data.DDT_AllWeaponsBit.stigmaXpToNextLevel": true,
-          "data.DDT_AllWeaponsBit.weaponManifestNumber": true,
-          "data.DDT_AllInventoryItemsBit": true,
-          "data.DDT_AllUnclaimedChestsBit": true,
-          "data.DDT_AllSceneEnactmentStatesBit": true,
-          "data.DDT_JourneyDataBit": true,
-          "data.DDT_GuideSystemBit": true,
-          "data.DDT_AllStoreItemsBit": true,
-          "data.DDT_AllFriendListsBit": true,
-          "data.DDT_SeasonalEventBit": true,
-          "data.DDT_ServerNotificationBit": true,
-          "data.DDT_CommunityGoalsBit": true,
-          "data.serverTime": true,
-          "log": true
-        },
-      },
-      { multi: true }
-    );
-    if (numAffected === 0) {
-      Logger.log("Error while migrating DLC characters");
-      throw new Error();
-    }
-    saveGames.compactDatafileAsync().then(() => {});
+
+    const fieldsToRemove = [
+      "data.DDT_AllPlayerAccountPointsBit": true,
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Cheerleader.points",
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_DollMaster.points",
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Jock.points",
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Outsider.points",
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Punk.points",
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Toad.points",
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Virgin.points",
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Werewolf.points",
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Eradicator.points",
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Nerd.points",
+      "data.DDT_AllLoadoutsBit.characterLoadouts.CT_Anomaly.points",
+      "data.DDT_AllLoadoutsBit.teenAffinities",
+      "data.DDT_AllLoadoutsBit.charXpLevelCosts",
+      "data.DDT_AllWeaponsBit.pointsByWeaponType",
+      "data.DDT_AllWeaponsBit.teenWeaponUnlockLevels",
+      "data.DDT_AllWeaponsBit.weaponXpToNextLevel",
+      "data.DDT_AllWeaponsBit.stigmaXpToNextLevel",
+      "data.DDT_AllWeaponsBit.weaponManifestNumber",
+      "data.DDT_AllInventoryItemsBit",
+      "data.DDT_AllUnclaimedChestsBit",
+      "data.DDT_AllSceneEnactmentStatesBit",
+      "data.DDT_JourneyDataBit",
+      "data.DDT_GuideSystemBit",
+      "data.DDT_AllStoreItemsBit",
+      "data.DDT_AllFriendListsBit",
+      "data.DDT_SeasonalEventBit",
+      "data.DDT_ServerNotificationBit",
+      "data.DDT_CommunityGoalsBit",
+      "data.serverTime",
+      "log"
```

Comment:
According to NEDB docks, those are keys, and values should be `true`for all of them